packages/twenty-front/src/modules/object-record/relation-picker/components/SingleEntitySelectMenuItemsWithSearch.tsx

```diff
@@ -35,13 +38,20 @@ export const SingleEntitySelectMenuItemsWithSearch = ({
 
   const showCreateButton = isDefined(onCreate) && searchFilter !== '';
 
+  // boardCardId indicates we're in a card in the opportunities page
```

Comment:
Could we translate this into a boolean variable ? Even if it's only `= isDefined(boardCardId)`

Reply:
And please feel free to do more of this adding context in naming, it's always appreciated.